### PR TITLE
chore: updates to prometheus docs

### PR DESCRIPTION
**Pull request by azhou-determined**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/integrations/prometheus.txt [hunk 1]
```diff
@@ -122,7 +125,7 @@ This example startup script includes the default setup docker commands provided
 
 #. Launch Prometheus with the provided `prometheus.yml
    <https://github.com/determined-ai/works-with-determined#observability-tools>`__ configuration
-   file.
+   file. This file needs a manual change to replace the placeholder master address.
```
Comment (shiyuann) [inline docs/integrations/prometheus.txt:128]:
```suggestion
   file. **This file needs a manual change to replace the placeholder master address.**
```

**Review by shiyuann — APPROVED** [review]

File: docs/integrations/prometheus.txt [hunk 2]
```diff
@@ -37,10 +37,13 @@ file.
 
 The Determined Prometheus endpoint is configured to work with `cAdvisor
 <https://github.com/google/cadvisor>`__ for CPU metrics and `dcgm-exporter
-<https://github.com/NVIDIA/dcgm-exporter>`__ (DCGM) for GPU metrics. Although other monitoring tools
-can be used with the setup, this guide describes only cAdvisor and DCGM tool configuration.
-Prometheus queries on metrics collected by other tools can differ, depending on the format and
-organization of the returned metrics.
+<https://github.com/NVIDIA/dcgm-exporter>`__ (DCGM) for GPU metrics. The default ports for cAdvisor
+and dcgm-exporter (8080 and 9400, respectively) are hardcoded into the Prometheus endpoint. If an
```
Comment (gh-determined-ai) [inline docs/integrations/prometheus.txt:41]: Change "The default ports for cAdvisor and dcgm-exporter (8080 and 9400, respectively) are ..."
to: "The default ports for cAdvisor, 8080, and dcgm-exporter, 9400, are ..."
Parentheses are not permitted. "Respectively" is too wordy.

File: docs/integrations/prometheus.txt [hunk 3]
```diff
@@ -37,10 +37,13 @@ file.
...
+agent is running on the same machine as the master, the master must be running on a port other than
+8080 in order for cAdvisor metrics to be scraped. Although other monitoring tools can be used with
```
Comment (gh-determined-ai) [inline docs/integrations/prometheus.txt:43]: In "... other than 8080 in order for ..." delete "in order". Unnecessary.

File: docs/integrations/prometheus.txt (hunk 1 above)
Comment (gh-determined-ai) [reply docs/integrations/prometheus.txt:128]: No! We don't use typography for syntax highlighting. Do not use `**`.
Everything in the doc is equally important or it wouldn't be there. Users need to read the doc, not scan for highlighted syntax. Similarly, we don't use `.. note::`

**Review by gh-determined-ai — APPROVED** [review]
